docs/connector-development/config-based/connector-builder-compatibility.md

```diff
@@ -0,0 +1,210 @@
+    - [Session authentication](#session)
+- [ ] [Is the data returned as JSON format?](#is-the-data-returned-as-json-format)
+- [ ] [If records are paginated, are they using one of the following mechanism?](#how-are-records-paginated)
+    - [Limit-offset](#limit-offset--offsetincrement-)
+    - [Page count](#page-count)
+    - [Link to the next page](#link-to-next-page--cursorpagination-)
+- [ ] [Do endpoints only require query parameters that are key-value pairs?](#does-the-integration-require-query-params-that-are-not-key-value-pairs)
+
+You can follow [this chart](https://whimsical.com/how-should-i-build-my-connector-YNXaNQhRuPuWy68Xg8Akhy) to guide you through the process.
+
+The sections below provide more details to help you answer the questions.
+
+## Is the integration a REST API returning a collection of records synchronously?
+
+Is the integration a REST API? In doubt, look for the word “REST” in the documentation.
+
+Rook at the shape of the resource path.
+
+Taking the [Congress API](https://api.congress.gov/#/bill) as an example,
+
+`GET /bill`
+
+Indicates the records can be retrieved by submitting a GET request to the /bill path.The sample response shows that the response returns a collection of records, so the Congress API is REST API returning a collection of records.
+
+Sample response:
+```
+{
+  "bills":[
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Became Public Law No: 117-108."
+      },
+      "number":"3076",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Postal Service Reform Act of 2022",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:27:05Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3076?format=json"
+    },
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Read twice. Placed on Senate Legislative Calendar under General Orders. Calendar No. 343."
+      },
+      "number":"3599",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Federal Rotational Cyber Workforce Program Act of 2021",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:41:50Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3599?format=json"
+    }
+  ]
+}
+```
+
+Some endpoints are parameterized. An example of such endpoint is the [Congress API’s bill by congress endpoint](https://api.congress.gov/#/bill/bill_list_by_congress).
+`GET /bill/:congress`
+
+These endpoints are also valid synchronous REST endpoints.
+
+This differs from the [Amazon Ads reports endpoint](https://advertising.amazon.com/API/docs/en-us/info/api-overview)), which returns a report ID, which will be generated asynchronously by the source. Because the reports need to be downloaded separately, this is not a synchronous REST API.
```

Comment:
```suggestion
This differs from the [Amazon Ads reports endpoint](https://advertising.amazon.com/API/docs/en-us/info/api-overview), which returns a report ID, which will be generated asynchronously by the source. Because the reports need to be downloaded separately, this is not a synchronous REST API.
```